src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -94,13 +95,20 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     });
   }
 
-  // Redirect to the logout page if an unauthorized API response was received.
+  // Log errors and info for unauthorized API responses
   function handleUnauthorizedAPIResponse(error) {
-    const errorStatus = error && error.response && error.response.status;
-    if (errorStatus === 401 || errorStatus === 403) {
+    const response = error && error.response;
```

Comment:
Maybe we want something like `logInfoForAPIResponse` in frontend-logging that could encapsulate this, like logAPIErrorResponse does?  (Not sure of the best name.)